### PR TITLE
feat : badge

### PR DESCRIPTION
**Pull request by pankaj-prog**

### CODE REVIEW

File: demo.html [hunk 1]
```diff
@@ -121,6 +121,54 @@ <h2>Floating button</h2>
...
+        <i class="fas fa-envelope"></i>
+        <span class="badge">5</span>
+      </div>
+      <div class="badge-container badge-icon-container">
+        <i class="fas fa-address-book"></i>
+        <span class="badge">9</span>
+      </div>
+      <h2>Badge on Avatar</h2>
+      <div
+        class="avatar avatar-lg avatar-round badge-container badge-avatar-container"
```
Comment (manojsarna) [inline demo.html:143]: check the closing the tag !!!!

File: src/components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+.badge-text-container .badge {
+  font-size: 0.8em;
+  top: -0.7em;
+  right: -0.7em;
+  width: 1.5em;
+  height: 1.5em;
+}
+
```
Comment (manojsarna) [inline src/components/badge/badge.css:29]: clean code.!!!

**Review by manojsarna — APPROVED** [review]
Good Work!! Good quality code. 

LGTM!!

GTM!!

File: src/components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  right: -0.5em;
+}
+
+.badge-avatar-container {
+  background: none;
+  padding: 0;
+}
+
+.badge-avatar-container .badge {
+  border: 3px solid white;
```
Comment (anshulraheja) [inline src/components/badge/badge.css]: Use variable for border color.

File: src/components/badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+.badge-text-container {
+  font-size: 1rem;
+  padding: 0.2em 0.5em;
+}
+
+.badge {
+  background: red;
+  color: white;
```
Comment (anshulraheja) [inline src/components/badge/badge.css]: use color variables

**Review by anshulraheja — APPROVED** [review]
Clean Code!!

File: src/components/badge/badge.css (hunk 4 above)
Comment (pankaj-prog) [reply src/components/badge/badge.css]: Yeah okay. i forget to change these.  I will update.

File: src/components/badge/badge.css (hunk 2 above)
Comment (pankaj-prog) [reply src/components/badge/badge.css:29]: Thank you.

**Pull request merged**